.gitignore

```diff
@@ -6,4 +6,7 @@ _build/
 dist/
 MANIFEST
 Axelrod.egg-info
-.hypothesis/eval_source/
+.hypothesis/
```

Comment:
I actually went the other way with this here: https://github.com/Axelrod-Python/Axelrod/commit/fd1b3bdf97369ff332f54fae167ccd14ee76ccf4

That follows https://hypothesis.readthedocs.org/en/latest/database.html#sharing-your-example-database which lets the hypothesis db get shared. 

Was there a reason why it perhaps wasn't working for you? Maybe the part about adding this to the .git/config (which can't be in the commit I linked to above):

```
[merge "hypothesisdb"]
    name = Hypothesis database files
    driver = python -m hypothesis.tools.mergedbs %O %A %B
```